src/analysis/normed_space/basic.lean

```diff
@@ -1007,6 +1082,17 @@ lemma summable_of_norm_bounded {f : ι → α} (g : ι → ℝ) (hg : summable g
   summable f :=
 by { rw summable_iff_cauchy_seq_finset, exact cauchy_seq_finset_of_norm_bounded g hg h }
 
+lemma summable_of_norm_bounded_eventually {f : ι → α} (g : ι → ℝ) (hg : summable g)
```

Comment:
Please add a docstring and mention limit comparison test there. It's not the same but very similar.

Reply:
I added a docstring, mentioning the "direct comparison test" which I think is more similar.